src/registrar/models/domain_request.py

```diff
@@ -100,8 +100,8 @@ class StateTerritoryChoices(models.TextChoices):
     class OrganizationChoices(models.TextChoices):
         """
         Primary organization choices:
-        For use in django admin
-        Keys need to match OrganizationChoicesVerbose
+        For use in the request experience
```

Comment:
Nitpick:  I still feel the sting of renaming from "domain application" to "domain request".  Better to be more verbose and say "domain request" (longer, more specific names is actually more robust).  Also, might not hurt to leave in "django admin" since it is still used for that, too?

Reply:
(In general, I have found code is easier to manage with consistent verbage than a mix of long/abbreviated forms.  I know this is just a comment, but I am having flashbacks...)